src/psyclone/tests/domain/lfric/arg_ordering_test.py

```diff
@@ -41,8 +41,8 @@
 import os
 import pytest
 
-from psyclone.domain.lfric import (KernCallArgList,
-                                   KernStubArgList, LFRicArgDescriptor)
+from psyclone.domain.lfric import (KernCallArgList, KernStubArgList,
```

Comment:
We could drop this modification too but I guess it doesn't matter. (I just like to reduce the chance of clashes.)

Reply:
Again fair point.